content/en/docs/usage/kube-csr.md

```diff
@@ -113,13 +113,55 @@ values that do not exist as `spec` or `status` fields on the
 CertificateSigningRequest resource. These fields are either set by the
 _requester_ or by the _signer_ as labelled below.
 
+Requester annotations:
+
 - `experimental.cert-manager.io/request-duration`: **Set by the requester**. Accepts
     a [Go time duration](https://golang.org/pkg/time/#ParseDuration) string
     specifying the requested certificate duration. Defaults to 90 days.
 
 - `experimental.cert-manager.io/request-is-ca`: **Set by the requester**. If set to
     `"true"`, will request for a CA certificate.
 
+- `experimental.cert-manager.io/private-key-secret-name`: **Set by the
+    requester**. Required only for the SelfSigned signer. Used to reference a
```

Comment:
Nit: could we state somewhere that in this doc 'signer' means the same as 'issuer/clusterisuer'? It is somehwat inferrable, but I imagine may still be confusing.